### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/reference/background.md`

### PR DESCRIPTION
**Pull request by lucasbalieiro**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 416facfddf0392ce4fe6767d2c39ae919fd9659a at: https://82eea486.fastapitiangolo.pages.dev

File: docs/pt/docs/reference/background.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Tarefas em Segundo Plano - `BackgroundTasks`
+
+Você pode declarar um parâmetro em uma *função de operação de caminho* ou em uma função de dependência com o tipo `BackgroundTasks`, e então utilizá-lo para agendar a execução de tarefas em segundo plano após o envio da resposta.
```
Comment (ceb10n) [inline docs/pt/docs/reference/background.md:3]: Most translations are translating _path operation function_ to _**função de operação de rota**_.

**Review by ceb10n — APPROVED** [review]
Great translation @lucasbalieiro  ✨✨

It would be nice to maintain the standard in _função de operação de rota_, but everything is correct.

File: docs/pt/docs/reference/background.md (hunk 1 above)
Comment (lucasbalieiro) [reply docs/pt/docs/reference/background.md]: Thanks! Translation updated in this commit [86f3a46](https://github.com/tiangolo/fastapi/pull/11849/commits/86f3a466a2ca233a85ea221ec509f41a2e860c94)

**Comment by tiangolo** [comment]
📝 Docs preview for commit 86f3a466a2ca233a85ea221ec509f41a2e860c94 at: https://e9d0bf6e.fastapitiangolo.pages.dev

**Review by Joao-Pedro-P-Holanda — APPROVED** [review]
Nice translation @lucasbalieiro! 🚀

**Review by ceb10n — APPROVED** [review]
🎉

**Review by nazarepiedady — APPROVED** [review]
Good, the translated content is great to be merged.

Thanks for your contribution.

**Review by marcelomarkus — APPROVED** [review]

**Comment by alejsdev** [comment]
Nice! Thanks to all for the contribution and help  :partying_face: :tada:

**Pull request merged**